app/views/tickets/index.html.haml

```diff
@@ -35,12 +35,11 @@
         .pull-right
           .btn-group-vertical
             = button_tag(type: 'submit', class: 'btn btn-success btn-lg') do
-              Support
+              Continue
               %i.fa.fa-shopping-cart
-            = link_to 'Continue without a Ticket!', conference_conference_registration_path(@conference.short_title),
-            class: 'btn btn-danger btn-sm'
+            = link_to 'Cancel registration', conference_conference_registration_path(@conference.short_title), method: :delete, class: 'btn btn-danger btn-sm'
   .row
     .col-md-13
       %p.text-muted.text-center
         %small
-          * Buying a ticket is not mandatory. Checkout will be at the conference registration.
+          * Buying a ticket is mandatory. Your registration will not complete until you buy a ticket.
```

Comment:
In this screen, you can select no tickets and click "continue"...
Do you think we should validate that at least 1 ticket was chosen before continuing?